.env.test

```
@@ -6,6 +6,7 @@ CREDENTIALS_ENCRYPTION_KEY="gWc408f8j3ZQUm5Ly4EXQC2EwFRf50AhePkmZKL5N9E="
 DB_NAME=":memory:"
 ENV="test"
 IN_TESTS='True'
+ZEAL_RAISE='True'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)